codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithyjava/nameresolver/AwsSdkNativeV2.java

```diff
@@ -96,26 +100,7 @@ private void checkForAwsServiceConstants() {
   }
 
   /** Validates that Polymorph knows non-smithy modeled constants for an AWS Service */
-  private static void checkForAwsServiceConstants(String namespace) {
-    boolean knowBaseException =
-      AWS_SERVICE_NAMESPACE_TO_BASE_EXCEPTION.containsKey(namespace);
-    if (!knowBaseException) {
-      throw new IllegalArgumentException(
-        "Polymorph does not know this service's Base Exception: %s".formatted(
-            namespace
-          )
-      );
-    }
-    boolean knowClientInterface =
-      AWS_SERVICE_NAMESPACE_TO_CLIENT_INTERFACE.containsKey(namespace);
-    if (!knowClientInterface) {
-      throw new IllegalArgumentException(
-        "Polymorph does not know this service's Client Interface: %s".formatted(
-            namespace
-          )
-      );
-    }
-  }
+  private static void checkForAwsServiceConstants(String namespace) {}
```

Comment:
Is there a reason to keep this?

Reply:
I had initially thought so, since although I've removed all the checks necessary there could be some in the future. But on second thought, I think those kinds of checks should be model validations instead anyway, so I've removed this.